### PR TITLE
Update Now Teach content

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/papQTf6k/7051

### CODE REVIEW

File: app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb [hunk 1]
```diff
@@ -9,7 +9,9 @@
     <p>You can chat to your dedicated adviser by phone, text or email.</p>
     <p>Find out more about <a href="/teacher-training-advisers">getting a teacher training adviser</a>.</p>
     <h3>Support from Now Teach</h3>
-    <p>If you have a degree, you may be able to get <a href="https://nowteach.org.uk/">support from the charity Now Teach</a>. They specifically help people get into teaching who are changing career.</p>
+    <h3>Support from Now Teach</h3>
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb:12]:
```suggestion
```

File: app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb [hunk 2]
```diff
@@ -9,7 +9,9 @@
...
+    <p>Now Teach is a charity that provides support to career changers. Their service specifically helps experienced professionals get into teaching and offers two years of one-to-one career change support as they train and beyond.</p>
+    <p>Find out more about<a href="https://nowteach.org.uk/">support from Now Teach</a>.</p>
```
Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb:14]:
```suggestion
    <p>Find out more about <a href="https://nowteach.org.uk/">support from Now Teach</a>.</p>
```

**Review by Sarah-DfE — COMMENTED** [review]
one teeny comment

File: app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb [hunk 3]
```diff
@@ -9,7 +9,8 @@
     <p>You can chat to your dedicated adviser by phone, text or email.</p>
     <p>Find out more about <a href="/teacher-training-advisers">getting a teacher training adviser</a>.</p>
     <h3>Support from Now Teach</h3>
-    <p>If you have a degree, you may be able to get <a href="https://nowteach.org.uk/">support from the charity Now Teach</a>. They specifically help people get into teaching who are changing career.</p>
+    <p>Now Teach is a charity that provides support to career changers. Their service specifically helps experienced professionals get into teaching and offers two years of one-to-one career change support as they train and beyond.</p>
```
[the remark this answers is not in the document]
Comment (gemmadallmandfe) [reply app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb:12]: No!

Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb:12]: Can we tweak slightly - it seems odd talking about `they` when normally we talk about `you`?

Comment (emmaachesongray) [inline app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb:12]:
```suggestion
    <p>Now Teach is a charity that provides support to career changers. Their service helps experienced professionals get into teaching and offers 2 years of one-to-one career change support as they train and beyond.</p>
```

**Review by Sarah-DfE — APPROVED** [review]
Looks good to me @emmaachesongray

**Pull request merged**